development.md

```diff
@@ -3,21 +3,32 @@ layout: page
-As soon as possible, the reviewer looks through the pull request, adds comments where necessary, and, according to the outcome of the review, approves it or requests changes.
+If no Issues are currently assigned to the team member, he picks up, from the 'Backlog' column, the one above all others (the currently most relevant), assigns it to him and, moves it to the `In progress` column.
 
-Once the review is completed, developer and reviewer meet (physically or remotely) and complete the review by going together through the comments.
+At this point, the team member creates locally an 'issue' branch named like the issue title and implements one after the other all the *Conditions of satisfaction* listed in the Issue using [Test-Driven Development](https://inforlife.github.io/process/test-driven-development.html).
 
-If the pull request contains any code that requires changes, it is updated according to the review before the branch is merged into **master** (and the issue marked as *reviewed*). Otherwise, the merge can be done as soon as the review is completed by the branch owner.
+The new code is pushed regularly to GitHub (at least one per day).
 
-We believe that code review is the single biggest thing we can do to improve the quality of the code we produce and, as very positive side effect, it also improves developers' skills.
+After the first push, the team member opens in GitHub a [Pull request](https://help.github.com/articles/using-pull-requests) to merge the issue branch into `master`.
 
-Note that code review may be skipped for features developed through [pair programming](https://en.wikipedia.org/wiki/Pair_programming).
+After every push to GitHub, the code goes through [CI](https://inforlife.github.io/process/ci.html), to ensure all the tests pass, and through Code Climate which looks for and reports quality issues.
 
-To make sure nobody introduces changes into production without review, we have defined the following rules.
+When the issue has been implemented in all its parts (all conditions of satisfaction are met), the code must be [reviewed](https://inforlife.github.io/process/code-review.html) before it can be merged into `master`. Thus, the branch owner, after making sure all the required checks pass, selects a reviewer (another member of the team).
 
-1. Nobody can add code directly to **master**, but only through the merge of a pull request.
+The branch owner is allowed to require a code review when Code Climate fails because of "false-positives", that are issues that the branch owner retains irrelevant. All relevant violations raised by Code Climate are addressed before starting the code review.
+
+As soon as possible, the reviewer goes through the Pull request.
+
+If during the review, the reviewer decides to make any changes to the code, instead of giving a status to the review, he assigns back the review to the branch owner for approval.
```

Comment:
Are you sure we can do so?

Reply:
What are your concerns? The branch owner cannot be a reviewer?

Reply:
I think so. See for example the current branch.
![schermata 2017-10-18 alle 10 17 23](https://user-images.githubusercontent.com/2420352/31707832-acf95ab0-b3ed-11e7-972b-bd6ac2e7314c.png)

Reply:
I see. So, I suppose the reviewer should not make any change since those changes can't be reviewed and by accident, something could be wrong. What do you think?

Reply:
I think this is the only possibility we have to be sure everything is reviewed.

Reply:
👍